src/components/Textarea/index.js

```diff
@@ -102,14 +104,16 @@ class Textarea extends Component {
 
         return (
             <StyledContainer className={className} style={style} id={id}>
-                <Label
-                    label={label}
-                    hideLabel={hideLabel}
-                    required={required}
-                    inputId={this.textareaId}
-                    readOnly={readOnly}
-                    id={this.getInlineTextLabelId()}
-                />
+                <StyledLabelContainer labelAlignment={labelAlignment} readOnly={readOnly}>
```

Comment:
since the `StyledContainer` use flex, you can use `align-self` directly in label thus avoiding creating a new container